src/analysis/inner_product_space/cross_product.lean

```diff
@@ -0,0 +1,181 @@
+/-
+Copyright (c) 2022 Heather Macbeth. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Heather Macbeth
+-/
+import analysis.inner_product_space.dual
+import analysis.inner_product_space.orientation
+
+/-! # The cross-product on an oriented real inner product space of dimension three
+
+This file gives a construction of the cross-product on an oriented real inner product space `E` of
+dimension three.
+
+## Implementation notes
+
+Notation `×₃` for `orientation.cross_product` should be defined locally in each file which uses it,
+since otherwise one would need a more cumbersome notation which mentions the orientation explicitly
+(something like `×₃[o]`).  Write
+```
+local infixl ` ×₃ `: 74 := o.cross_product
```

Comment:
I don't know if `localized`, `open_locale` are powerful enough but it would be nice if we set things up so that the user could just write something like `open_locale cross_product using o`.

Probably best not to bother here, especially with the port to Lean 4 on the horizon.